client/my-sites/plugins/plugin.jsx

```diff
@@ -385,6 +386,7 @@ const SinglePlugin = React.createClass( {
 
 		return (
 			<MainComponent>
+				<QuerySites allSites />
```

Comment:
Yeah, I discussed this a bit earlier with Javi in https://github.com/Automattic/wp-calypso/pull/11541#discussion_r103473313

Doing this here is ok since the refactor costs would be be too high and undesirable ( going back towards an old style), but let me verify if we need the new usage, or if we call this in the section someplace.